emission_events/emission_events/settings/local.py

```diff
@@ -24,9 +24,14 @@
 
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 DATABASES = {
-    'default': dj_database_url.config(
-        default=os.environ.get('EMISSIONS_DATABASE')
```

Comment:
The current scraper probably depends on this, so we probably shouldn't just rip it out.